online-shop/src/app/product-list/product-list.component.scss

```diff
@@ -0,0 +1,35 @@
+table {
+  margin-right: auto;
+  margin-left: auto;
+  font-size: 1.25rem;
+  border: 1px solid black;
+  border-collapse: collapse;
+  text-align: center;
+  background-color: antiquewhite;
+}
+
+th {
```

Comment:
in this case th and td have the same style. You have two options:
the, td {
*add style here
}

or

write a CSS class that contains the styling you want, and apply it to the the and td